.github/file-paths.yaml

```diff
@@ -68,7 +68,7 @@ sources: &sources
   - *backend_sources
 
 e2e_specs: &e2e_specs
-  - "**.cy.*.js"
+  - "**/*.cy.*.js"
```

Comment:
The same principle applies:
https://github.com/metabase/metabase/pull/28447/files#r1111982837